src_updater/updater.py

```diff
@@ -194,7 +194,7 @@ def do_update(self):
             mkdir_p(scratch)
             try:
                 self.update_step = UpdateStep.DOWNLOAD
-                url = 'https://github.com/Hyphen-ated/RebirthItemTracker/releases/download/' + self.latest_version + '/Rebirth.Item.Tracker-' + self.latest_version + ".zip"
+                url = 'https://github.com/rchardon/RebirthItemTracker/releases/download/' + self.latest_version + '/Rebirth.Item.Tracker-' + self.latest_version + ".zip"
```

Comment:
In a pull request, you shouldn't change references pointing to the main repository. This cannot be merged while this is present.

Also, sneakingly changing the update source is a pretty malicious action.